newIDE/app/src/ObjectsRendering/PixiResourcesLoader.js

```diff
@@ -238,15 +239,14 @@ export default class PixiResourcesLoader {
         )
       );
 
-    const fullUrl = ResourcesLoader.getResourceFullUrl(project, resourceName, {
+    let fullUrl = ResourcesLoader.getResourceFullUrl(project, resourceName, {
       isResourceForPixi: true,
     });
     if (!fullUrl) {
-      return Promise.reject(
-        new Error(
-          `The resource called ${resourceName} was no found.`
-        )
+      console.warn(
+        `The resource called ${resourceName} was no found.\nThe default bitmap font will be used.`
       );
+      return;
```

Comment:
It's generally a bad practice to "maybe return a promise, but maybe undefined". Either your function is asynchronous and will *always* return a promise, or it's not, and it will *never* return a promise.

Here your function is asynchronous. It returns a promise. So even if you want to bail out, return either:
* A promise with `null` (`Promise.resolve(null)`) if it's expected that you can get null.
* Or `Promise.reject` if the caller must use a `catch` to handle the error.